sympy/functions/special/delta_functions.py

```diff
@@ -67,28 +67,30 @@ def eval(cls, arg, k=0):
         if arg.is_positive or arg.is_negative:
             return S.Zero
 
-    def simplify(self, x):
-        """simplify(self, x)
+    def _eval_expand_DiracDelta(self, **hints):
```

Comment:
I think this should be:

```python
def _eval_expand_diradelta(self, **hints):
```

if we want to follow pep8 and the current conventions in the codebase.